compiler/rustc_hir_typeck/messages.ftl

```diff
@@ -79,6 +79,11 @@ hir_typeck_field_multiply_specified_in_initializer =
     .label = used more than once
     .previous_use_label = first use of `{$ident}`
 
+hir_typeck_fn_item_to_variadic_function = can't pass a function item to a variadic function
+    .suggestion = use a function pointer instead
+    .help = a function item is zero-sized and needs to be casted into a pointer-sized function pointer to be used in FFI
```

Comment:
also, the past participle of cast is ["cast"](https://en.wiktionary.org/wiki/cast#Verb)

```suggestion
    .help = a function item is zero-sized and needs to be cast into a function pointer to be used in FFI
```

Also, the fact that it's zero-sized doesn't seem to me to be the most important part of this issue. The issue here is more that the *intention* is wrong, since you're not passing a function pointer like you expect.